src/components/Users/UsersContainer.jsx

```diff
@@ -1,34 +1,122 @@
 import React from "react";
 import { connect } from "react-redux";
+import * as axios from "axios";
 
 import {
-  updateUnfollowStatusAC,
-  updateFollowStatusAC,
-  setUsersAC,
+  follow,
+  unfollow,
+  setUsers,
+  setCurrentPage,
+  setTotalUsersCount,
+  toggleIsFetching,
 } from "../../redux/actionCreators";
 
-import Users from "./Users";
+import { Users } from "./Users";
+import { Preloader } from "../common/Preloader/Preloader";
+import photo000 from "../../assets/images/photo000.png";
 
-const mapStateToProps = (state) => {
-  return {
-    users: state.findUsersPage.users,
+import styles from "./Users.module.css";
+
+class APIUsersContainer extends React.Component {
+  componentDidMount() {
+    this.props.toggleIsFetching(true);
+    axios
+      .get(
+        `https://social-network.samuraijs.com/api/1.0/users?page=${this.props.currentPage}&count=${this.props.pageSize}`
```

Comment:
Consider storing the API url somewhere in the "constants.js" at the core app level.